Explorer/Assets/DCL/Character/CharacterMotion/Animation/ApplyAnimationMovementBlend.cs

```diff
@@ -55,5 +55,16 @@ private static int GetMovementBlendId(Vector3 velocity, MovementKind speedState)
                        _ => 0,
                    };
         }
+
+        public static int GetMovementBlendId(MovementKind speedState)
+        {
+            return speedState switch
+                   {
+                       MovementKind.Walk => 1,
+                       MovementKind.Jog => 2,
+                       MovementKind.Run => 3,
+                       _ => 0,
+                   };
+        }
```

Comment:
What's the purpose of this method? its duplicates another one

Reply:
the problem is that the other one required a Vector3 and if the vector was 0 it returned 0 xD
but this is not in use in this branch anyway, so I can remove it!

Reply:
okay Removed that :)